react-components/src/hooks/use3dRelatedDirectConnections.ts

```diff
@@ -21,12 +21,14 @@ export function use3dRelatedDirectConnections(
       });
 
       const view = views.items[0].inspectionResults.involvedViewsAndContainers.views[0];
-      const newInstance = await fdmSdk.getByExternalIds<Record<string, unknown>>(
-        [{ instanceType: 'node', ...assertedInstance }],
-        view
-      );
+      const instanceContent = (
+        await fdmSdk.getByExternalIds<Record<string, unknown>>(
+          [{ instanceType: 'node', ...assertedInstance }],
+          view
+        )
+      ).items[0];
```

Comment:
Should we add try/catch or something similar here in case instance is missing to not have an error?